e2e/libs/volume/rest.py

```
@@ -231,6 +231,23 @@ def get_replica_name_on_node(self, volume_name, node_name):
                 if r.hostId == node_name:
                     return r.name
 
+    def wait_for_replica_count(self, volume_name, node_name, replica_count):
+        for i in range(self.retry_count):
+            running_replica_count = 0
+            volume = get_longhorn_client().by_id_volume(volume_name)
+            for r in volume.replicas:
+                if node_name and r.hostId == node_name and r.running:
+                    running_replica_count += 1
+                elif not node_name and r.running:
+                    running_replica_count += 1
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)